### PR TITLE
Add message on final peg-out tx screen

### PR DESCRIPTION
**Pull request by lserra-iov**
Added a message at the end of the peg-out creation process with a link to the documentation informing how to get the funds in BTC.

### CODE REVIEW

File: src/store/constants.ts
Comment (julia-zack) [inline src/store/constants.ts:1]: i'll change "DOCS" for "DOCUMENTATION"

**Review by julia-zack — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Pull request merged**